src/lib/config.ts

```diff
@@ -130,6 +130,7 @@ export const JSON_WEB_TOKEN_EXPIRY_MINUTES = intVar(
 	'JSON_WEB_TOKEN_EXPIRY_MINUTES',
 );
 export const JSON_WEB_TOKEN_SECRET = requiredVar('JSON_WEB_TOKEN_SECRET');
+export const LOGS_HOST = optionalVar('LOGS_HOST', '');
```

Comment:
```suggestion
export const LOGS_HOST = optionalVar('LOGS_HOST');
```
Nit - probably don't need to set an empty default value.